features/user_sessions.feature

```diff
@@ -1,37 +1,17 @@
 Feature: User sessions
 
-  @javascript
-  Scenario: Already logged-in user visits login page
-    Given I am logged in with email "affiliate_admin@fixtures.org"
-    And I go to the login page
-    Then I should see "Contact Information"
```

Comment:
Is there a reason this was deleteted? This is a valid scenario that just needs updating to reflect the correct behavior. This is a super admin who is already logged in, so when they visit the login page again, they should be on the super admin landing page:
`Then I should be on the admin home page`

Reply:
Restored.